Collaboration2/Collaboration2/CloudAnimationView.swift

```diff
@@ -0,0 +1,88 @@
+//  CloudAnimationView.swift
+//  Collaboration2
+//
+//  Created by Akaki Titberidze on 12.06.24.
+//
+
+import SwiftUI
+
+struct CloudAnimationView: View {
+    @State private var clouds: [Cloud] = []
+    var cloudSetHalf: Bool
+    
+    init(cloudSetHalf: Bool = false) {
+        self.cloudSetHalf = cloudSetHalf
+    }
+    
+    var body: some View {
+        GeometryReader { geometry in
+            ZStack {
+                ForEach(clouds.indices, id: \.self) { index in
+                    let cloud = clouds[index]
+                    let width = CGFloat.random(in: 50...140)
+                    let size = CGSize(width: width, height: width * 0.6)
+                    let imageName = Bool.random() ? "cloud1" : "cloud2"
+                    CloudView(startingAt: cloud.position, duration: cloud.duration, screenSize: geometry.size, size: size, imageName: imageName, direction: cloud.direction)
```

Comment:
for functions or initialisers with this many parameters write every parameter on the new line